app/components/InstanceAutoRestartPopover.tsx

```diff
@@ -69,7 +69,7 @@ export const InstanceAutoRestartPopover = ({ instance }: { instance: Instance })
                 </Badge>
               ))
               .with('best_effort', () => <Badge>best effort</Badge>)
-              .with(undefined, () => <Badge color="neutral">Default</Badge>)
+              .with(P.nullish, () => <Badge color="neutral">Default</Badge>)
```

Comment:
Jerk!

Reply:
Confirmed a test fails with a `null` policy without the fix, pass with the fix.

<img width="881" alt="Screenshot 2025-05-06 at 5 56 36 PM" src="https://github.com/user-attachments/assets/f3bb33cf-810d-4fea-9fa2-434600766e14" />